### ISSUE TITLE
Bar chart

### ISSUE THREAD

**Issue by abhi1693** [influxdata/giraffe#610]
Ability to display flux response using a bar chart

**Comment by TCL735** [comment]
This is similar to #379

**Comment by PyAntony** [comment]
Yes please, we need bar chart and pie chart. I don't want to use an additional library, it's such a hassle.

**Comment by adtech-scott** [comment]
no movement on this? i agree, this is pretty annoying, histogram gets so close, yet stays so useless.

**Comment by ErikBooij** [comment]
Hey folks 👋 We're almost a year and a half further since the last question on progress and still... nothing? It honestly feels insane that an otherwise fabulous piece of software that tries to compete with the likes of Grafana doesn't have such incredibly basic functionality and seems to be in absolutely no rush to acknowledge the fact that this is a problem, let alone fix it.

**Comment by elliotfontaine** [comment]
For real...

**Comment by abhi1693** [comment]
I'm afraid so
